packages/alpinejs/src/directives/x-transition.js

```diff
@@ -173,7 +173,7 @@ window.Element.prototype._x_toggleAndCascadeWithTransitions = function (el, valu
                     let carry = Promise.all([
                         el._x_hidePromise,
                         ...(el._x_hideChildren || []).map(hideAfterChildren),
-                    ]).then(([i]) => i())
+                    ]).then(([i]) => i && i())
```

Comment:
```suggestion
                    ]).then(([i]) => i?.())
```

Not sure if we want to reopen this can of worms with people using decade old webpack versions that can't support optional chaining, but we can dream.

Reply:
Thanks for the suggestion. Support for optional chaining seems to be wide-spread now: https://caniuse.com/mdn-javascript_operators_optional_chaining
And it seems that we're already using this feature in this code-base, so let's use this here.

https://github.com/alpinejs/alpine/blob/0d5a190bc674c0b141c7a534b5b1e132c6510359/packages/alpinejs/src/scope.js#L74
https://github.com/alpinejs/alpine/blob/0d5a190bc674c0b141c7a534b5b1e132c6510359/packages/ui/src/combobox.js#L27